TelepresenceBot/server-unit/core/loggingClient.js

```diff
@@ -6,7 +6,11 @@ LoggingClient.prototype.emit = function(event, dataToEmit) {
     if(this.client != undefined) {
         this.client.emit(event, dataToEmit);
     }
-    console.log("\nEvent Emitted: " + event + "\nData Emitted: " + dataToEmit + "\n");
+    console.log("\nEvent Emitted: " + event);
+
+    if(dataToEmit != undefined && dataToEmit.length > 0) {
```

Comment:
Just split this up a little bit because sometimes I would log when we actually have no data to log 😄